Content.Shared/Buckle/SharedBuckleSystem.Buckle.cs

```diff
@@ -354,7 +354,7 @@ public bool TryBuckle(EntityUid buckleUid, EntityUid userUid, EntityUid strapUid
 
        ReAttach(buckleUid, strapUid, buckleComp, strapComp);
        SetBuckledTo(buckleUid,strapUid, strapComp, buckleComp);
-       _audioSystem.PlayPredicted(strapComp.BuckleSound, strapUid, buckleUid);
```

Comment:
Maybe add a TODO comment at the top of the function or something saying that this needs to be fixed if buckling ever actually gets predicted.

Reply:
done, though buckling should be completely predicted currently